app/models/authenticator/ldap.rb

```diff
@@ -34,6 +34,9 @@ def find_or_create_by_ldap(username)
       user = userprincipal_for(username)
       return user unless user.nil?
 
+      user = autocreate_user(username)
+      return user unless user.nil?
```

Comment:
Here is the complete method:

```ruby

    def find_or_create_by_ldap(username)
      username = miq_ldap.fqusername(username)
      user = userprincipal_for(username)
      user = autocreate_user(username) if user.nil?
      return user unless user.nil?

      raise _("Unable to auto-create user because LDAP bind credentials are not configured") unless authorize?

      create_user_from_ldap(username) do |lobj|
        groups = match_groups(groups_for(lobj))
        if groups.empty?
          raise _("Unable to auto-create user because unable to match user's group membership to an EVM role")
        end
        groups
      end
    end
```